app/android/app/build.gradle

```diff
@@ -144,7 +147,7 @@ android {
         versionCode 1
         versionName "1.0"
         buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()
-        manifestPlaceholders = [auth0Domain: "{DomainName}", auth0Scheme: "${applicationId}"]
+        manifestPlaceholders = [auth0Domain: project.env.get("DOMAIN_NAME"), auth0Scheme: "${applicationId}"]
```

Comment:
Awesome! Love this change, won't have to keep a `git stash` around anymore for this haha.